### DISCUSSION TITLE
关于b站视频下载与idm自动配合

### DISCUSSION THREAD

**Discussion by huanmoruyan** [the1812/Bilibili-Evolved#1523] [功能建议 / Ideas]
我在用这个插件的时候,发现真的下载视频的功能太棒了!但是可惜的一点是,我不想(懒)去一步一步地点击按钮来下载视频,而是直接在idm提供的下载框中便可以看到flv的视频内容. 可否提供一个功能能让idm的悬浮窗口可以直接下载
![image](https://user-images.githubusercontent.com/59784741/106556494-adaa1900-655a-11eb-9efd-842897090994.png)
视频的清晰度就提供最好的和目前所使用的
以上

**Comment by Geeyun-JY3** [comment]
重复：#1404
